src/main/java/org/sagebionetworks/web/client/widget/search/SynapseSuggestBoxViewImpl.java

```diff
@@ -29,9 +29,10 @@ public SynapseSuggestBoxViewImpl(SynapseAlert synAlert) {
 	@Override
 	public void configure(SynapseSuggestOracle oracle) {
 		final TextBox suggestTextBox = new TextBox();
-		suggestTextBox.getElement().setAttribute("name", "address");
+		suggestTextBox.getElement().setAttribute("name", "synapse-user-search");
 		suggestBox = new SuggestBox(oracle, suggestTextBox, new SynapseSuggestionDisplay());
 		suggestBox.getValueBox().addStyleName("form-control");
+		suggestBox.getValueBox().getElement().setAttribute("autocomplete", "off");
```

Comment:
try to turn off the browser autocomplete entirely for this input box (since the user needs to select an item from the dropdown UI for it to function properly)